packages/file/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: file
-version: 5.1.0
+version: 5.2.0-dev
```

Comment:
It's probably fine to just bump this to 5.2.0. This package isn't particularly active, I don't know that there's any other pending work that would be waiting for releasing this once it's done.

Reply:
Okay.  I wasn't sure if there was any concern about bumping the `y`-version now-ish and presumably again when `package:file` is updated for NNBD, which isn't too far away.